src/ctia/schemas/core.clj

```diff
@@ -20,196 +20,225 @@
              [schema :as f-schema]
              [spec :as f-spec]
              [utils :as fu]]
-            [schema.core :refer [Str Bool] :as sc]))
+            [schema.core :refer [Str Bool] :as sc]
+            [schema-tools.core :as st]))
+
+(sc/defschema ACLStoredEntity
+  (st/optional-keys
+   {:groups [Str]
```

Comment:
What is the difference between `:groups` and `authorized_groups`? If `:groups` is related to the owner, could you rename it to `owner_groups`?